### ISSUE TITLE
Setting values on a RangeView Object doesn't work as expected

### ISSUE THREAD

**Issue by misaunde** [OfficeDev/office-js#226]
## Expected Behavior

**When a column is filtered**, I can read it's values with:
`const range = context.workbook.getSelectedRange().getUsedRange().getVisibleView();
range.load("values");`

Similarly, I'd like to be able to write values to the visible cells in the column with:
`var visibleRange = context.workbook.getSelectedRange().getUsedRange().getVisibleView();
visibleRange.values = rangeValuesToSetVisible;`

Reason I would expect this to work: https://stackoverflow.com/a/45356860/3806701

## Current Behavior

Screenshot of unfiltered data:
![image](https://user-images.githubusercontent.com/10051350/44366878-577cb480-a48b-11e8-92ff-30e7f298802c.png)

Screenshot of filtered data:
![image](https://user-images.githubusercontent.com/10051350/44366905-6d8a7500-a48b-11e8-8952-08dec271c582.png)

Reading the values in column B with `.getVisibleView()` returns (as expected):
`[[1],[1],[1]]`

Trying to write different values with `.getVisibleView()` of same shape:
`[[15],[15],[15]]`

Gives me this error:
`InvalidArgument: The number of rows or columns in the input array doesn't match the size or dimensions of the range.`

It looks like the RangeView Object contains the cell addresses of the visible cells, and I could manually go through and write the values I choose; however, I think it's confusing and inconsistent to read values one way and write them another.

## Your Environment

* Platform [PC desktop, Mac, iOS, Office Online]: pc
* Host [Excel, Word, PowerPoint, etc.]: Excel 2016
* Office version number: Build 10325.20082
* Operating System: Windows

**Comment by XiangGaoMSFT** [comment]
Thanks for reporting it. I tried on Excel 2016 but can't repro it.

It may be related to your current selection.

Could you import [this gist](https://gist.github.com/XiangGaoMSFT/966b55acb05712be49222010b96475e8) to Script Lab and check whether `visibleRange.cellAddresses` has the same row/column count with `valuesToSet`?
If yes, could you provide Script Lab console's output to help us to investigate?

**Comment by misaunde** [comment]
Ok wow, fantastic, that worked. Lol, I've tried on 2 separate occasions to get this working, without success. My co-worker also manually walked through the cellAddresseses because it didn't appear to work for them the first time either. After playing with the gist though, I got my code working in 5 minutes and was left scratching my head why it didn't work before.

Anyways, there doesn't appear to be a .getCell() method on a RangeView object which seems strange to me as I'd like to iterate over my filtered range and apply formatting (fill color). Any suggestions on how to do that?
